handlers/secrets.go

```diff
@@ -75,3 +81,86 @@ func makeSecretsArray(c *client.Client, secretNames []string) ([]*swarm.SecretRe
 
 	return values, nil
 }
+
+func secretGetHandler(c *client.Client, w http.ResponseWriter, r *http.Request) {
+	secrets, err := c.SecretList(context.Background(), types.SecretListOptions{})
```

Comment:
How are we going to annotate / label secrets so that we don't get secrets which are not part of OpenFaaS?

Reply:
In the current Operator PR, a Stefan added a label `owner = openfaas` 

https://github.com/openfaas-incubator/openfaas-operator/pull/70/files#diff-78dbaab3a94c175dd3c0e4084e204125R25

Reply:
Sounds good to me. Does the API support that?

Reply:
Yes, the [`SecretSpec`](https://godoc.org/github.com/docker/docker/api/types/swarm#SecretSpec) includes the [`Annotations` struct](https://godoc.org/github.com/docker/docker/api/types/swarm#Annotations)
```golang
type Annotations struct {
    Name   string            `json:",omitempty"`
    Labels map[string]string `json:"Labels"`
}
```

Are there any standard labels that are recommended in Swarm? I can't find a doc like there is for k8s: https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/

Reply:
👍 @bartsmykla please can you add a label?

@stefanprodan I think we should be using `com.openfaas` as a prefix

Reply:
I will!

Reply:
Done